### DISCUSSION TITLE
Welcome to Anti-Scam-Bot-Link-List!

### DISCUSSION THREAD

**Discussion by 2Lost4This** [AntiScamBot/Anti-Scam-Link-List#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by InvalidLenni** [comment]
The hi-chain

**Reply by 2Lost4This** [reply]
Hi

**Reply by InvalidLenni** [reply]
Hi

**Reply by 2Lost4This** [reply]
Hi

**Comment by KingCh1ll** [comment]
pog

**Reply by InvalidLenni** [reply]
Yeah!

**Reply by 2Lost4This** [reply]
> Yeah!

👀 Lets see how this will work

**Reply by InvalidLenni** [reply]
> > Yeah!
> 
> 👀 Lets see how this will work

fine. I found 1 - 3 typos btw

**Reply by InvalidLenni** [reply]
but idk more where

**Reply by KingCh1ll** [reply]
> > > Yeah!
> > 
> > 
> > 👀 Lets see how this will work
> 
> fine. I found 1 - 3 typos btw

typos?

**Reply by 2Lost4This** [reply]
👀
